### ISSUE TITLE
MrKnow URL resolver update failed

### ISSUE THREAD

**Issue by BankruptEd** [mrknow/filmkodi#758]
## Please follow the guide below / Podążaj za przewodnikiem
- Put an "x" into all boxes [ ] relevant to your *issue* (Wstaw "x" w każde pole [ ], które dotyczy twojego zgłoszenia)
- Use *Preview* tab to see how your issue will actually look like (Użyj *Perview* aby sprawdzić, czy wszystko wygląda dobrze)

---
## Kodi & OS
- Kodi verison: __x___
- OS version: _____

---
## Addon / Wtyczka
- [ ] Filmy online - Mrknow
- [ ] MrknowTV
- [ x] Specto
- [ ] FanFILM
- Version / Wersja: _____

---
## Issue / Zgłoszenie
- [ ] Bug report / Błąd
- [ ] Site request / Prośba o nową stronę
- [ ] Feature request / Prośba o nową funkcję
- [ ] Question / Pytanie
- [x ] Other / Inne

---
## Descryption / Opis
Put here every imported thing such logs, how to reproduce etc. <br/>
Wstaw tutaj każdą ważna rzecz, taką jak logi, jak wywołać błąd itp.
 Just letting you know that I was watching when this latest URL resolver update came though, it failed. It appears to have had some serious effects as well. Programs I watch such as How the Universe Works (a semi popular show) now only displays a few links, all of which don't work. And much.more popular shows such as The Simpsons now display maybe ten links as opposed to the usual dozens of links. They all seem to be HQ links, with no MQ or LQ links to choose from. So I think we may have a problem here.

**Comment by BankruptEd** [comment]
I'm surprised no one else has mentioned this yet. Hope it's not just me. I'm having to watch a large portion of my TV viewing (all except the newest, most popular shows) on Exodus. I just don't get links or links that work for anything other than the newest, most popular shows, and I don't get many links for those even. I wouldn't know how to rectify a MrKnow url resolver issue. Hope you can help. I know it's not even been 24 hours since I posted this, so I'm not trying to rush anything, just thought I would give an update.

**Comment by mslavishlyfe** [comment]
I'm having the same problem. He said that he would fix it 4 months ago. Smh. How do I uninstall all of his mess since he can't fix it??

**Comment by BankruptEd** [comment]
Really eh? That blows. Mine was fine til a few days ago, but it's bad. I
really can't watch most of what I used to watch on Specto.

**Comment by BankruptEd** [comment]
MrKnow, is there anything you can tell me about this? Is it likely to fix itself with next update? Or will I have to delete everything and start over? I would really like to fix this so I can keep my viewing within Specto where all my favorites are.

**Comment by BankruptEd** [comment]
MrKnow I'm pleading with you, please, please give me some guidance on this issue. As things currently stand, Specto is lost to me. Almost completely useless. I don't want to go elsewhere, this is my favorite, and in my opinion, THE best add-on there is when it's working. I'm not very computer savvy and I don't have a clue what steps to take to rectify this url resolver issue.

**Comment by BankruptEd** [comment]
Hi MrKnow. Hope you're well. Let me start by saying, I know you're busy. You have lots of issues to deal with in respect to Specto, and no doubt a life outside of Specto also demanding your time. That said, I'm going to respectfully ask again, if you could provide me with some information on what a person can do to fix a broken MrKnow url resolver? Right now Specto is not much use to me, and that's extremely disappointing. I LOVE SPECTO. If you can't fix it, could you suggest anything I could do? 

1 - will sitting tight and waiting for another url resolver update work? 

2 - When is the next url resolver update rolling out? I don't want to start uninstalling things until Specto has had the opportunity to fix itself with an update.

3 - If I have to uninstall, would I have to uninstall the repository as well?

4 - Is there anything else, anything at all that you can inform me of that might help me resolve this?

I know my last issue about the changelog was kind of trivial, and I was really just offering my opinion, and I will refrain from posting such issues in the future, but this one is a big, serious issue for me and I really need your help.

Thanking you in advance for this and thanks again for all your past efforts...

BankruptEd

**Comment by azrin1972** [comment]
Hi,

I'm just wondering,

Why is the URLresolver of yours is not working? I just updated mine and all seems to work fine. 

Can you share what machine configuration are you on. May be I can help you a bit.

**Comment by BankruptEd** [comment]
I run and inl3d ultra android box. I would have to check the version (not
at home right now) using the latest Kodi, the latest Specto. I was actually
in Specto when the update came through, and it simply said MrKnow url
resolver update failed. Ever since then, it can't find any sources for a
whole multitude of shows. And only seems to find HD and HQ sources when it
does find some. And this only seems to pertain to the newest, most popular
shows. So I just don't have a clue what I should do.

**Comment by azrin1972** [comment]
Any particular movies or shows that I can help to test?

**Comment by BankruptEd** [comment]
Well off the top of my head, "how the universe works" and "autopsy" are two
of them. These are shows that I was working my way through the seasons on
Specto, but for which I now get no sources at all. On the other end of
things, The Simpsons is a show that like with most people, I used to get
oodles of sources, in the hundreds sometimes. Now I might get 15. So that
is a good example of what's going on.

**Comment by BankruptEd** [comment]
I can give you lots more if you want, just not sure how many you would
like. Say so, and I can get tons more examples.

**Comment by BankruptEd** [comment]
Here's an interesting thing. For the newest Simpsons, I get no sources at
all. The show just before it, I get like five sources. It's like I'm
getting no new sources, and the old ones are running out kind of. I'm
getting less all the time. Eventually none I'm guessing.

**Comment by azrin1972** [comment]
just tested simpson season 28, s28e01 gives me 167 and s28e02 gives me 200+ links

Autopsy however only gives me between 8-38 links

For me as long as I got links and is playable, its good enough for me.

May be what you can do is to of to Specto->Tools and elect clear cache and clear sources.

**Comment by BankruptEd** [comment]
I cleared sources once already but not cache I don't think. I will give it
a shot. That's the only thing I don't like about kodi, I'm always kind of
living in fear of these problems cause I just don't know enough about it
and it's hard to find people willing to help like yourself and even then it
can still be difficult.

Ok I did it and still no go. I keep hoping another url resolver update will
come through and fix it but I haven't seen any Specto updates come through
some it happened. Hope thats not compromised too. Well I appreciate you
efforts anyhow.

**Comment by BankruptEd** [comment]
To mslavishlyfe - not sure if your problem is exactly the same as mine, but I did manage to get my url resolver up and running. Had to go to "add-on information" then "update" then obviously to MrKnow update selection. After I did this it showed (mrknow url resolver enabled) I guess the failure had shut it down altogether. So u might wanna try that even if you do auto updates as I do. It still fixed it up for me.
